blog/2020-09-24-debezium-1-3-cr1-released.adoc

```diff
@@ -0,0 +1,69 @@
+= Debezium 1.3.0.CR1 Released
+ccranfor
+:awestruct-tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, outbox ]
+:awestruct-layout: blog-post
+
+I'm very happy to announce the release of Debezium *1.3.0.CR1*!
+
+As we approach the final stretch of Debezium 1.3 Final,
+we took this opportunity to add delegate converter support for the `ByteBufferConverter` and introduce a `debezium-scripting` module.
+In addition, there's also a range of bug fixes and quite a bit of documentation polish;
+overall, not less than https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.3.0.CR1%20ORDER%20BY%20issuetype%20DESC&startIndex=20[15 issues] have been resolved for this release.
+
+== ByteBufferConverter improvements
+
+In order to fix a recent report (https://issues.jboss.org/browse/DBZ-2396[DBZ-2396]),
+the `ByteBufferConverter` can now be configured to delegate event serialization to an additional converter.
```

Comment:
Can you add a sentence explaining _why_ one would want to use that converter in the first place (outbox with existing Avro data in payload column) and also add a link to the right section of the docs?